internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -259,6 +263,11 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
         serverPropsBuilder.put(GRPC_SERVER_WORKER_THREAD_COUNT, numGrpcWorkerThreads);
       }
 
+      if (isPlainTableEnabled) {
+        serverPropsBuilder.put(ROCKSDB_PLAIN_TABLE_FORMAT_ENABLED, true);
+        serverPropsBuilder.put(ROCKSDB_OPTIONS_USE_DIRECT_READS, false); // Required by PlainTable format
+      }
```

Comment:
Thanks, it is good to change these together... although this makes me wonder whether we need the direct reads config at all, or if we should always set it according to the PT config in the main code also. Anyway, it's fine to leave it as is in this PR.

Reply:
I have to admit that I am not familiar with this config, but I searched PROD config, it is tightly coupled with the PT config in a few clusters. I'd suggest let's keep it as a bundle, and if we think it is no longer the case, we can change it.